src/NBomber/Infra/Dependency.fs

```diff
@@ -53,11 +56,25 @@ module ProgressBar =
 
 module Logger =
 
-    let initLogger (appType: ApplicationType) =
-        Log.Logger <- 
-            match appType with            
-            | Console -> LoggerConfiguration().WriteTo.Console().CreateLogger()
-            | _       -> LoggerConfiguration().CreateLogger()
+    let createLogger (config: LoggerConfiguration) =
+        config.CreateLogger()
+
+    let appendConsoleOutput appType (config: LoggerConfiguration) =
+        match appType with
+        | Console -> config.WriteTo.Console()
+        | _       -> config
+
+    let appendFileOutput (logSetting: LogSetting option) (config: LoggerConfiguration) =
+        match logSetting with
+        | Some lg when not << String.IsNullOrWhiteSpace <| lg.LogToFile ->

```

Comment:
seriously I don't know this operator <<

Reply:
Just some opinions from me.
On the one side this operator is not that uncommon to not knowing them - just an opposite of `>>` - applying the second function and then first. :)
Not that I see any big problems with it, But personally I too don't like operators which work in the opposite directioin like this or `<|`. or a friends. In this particular case it reads a lot more smoothely with just two forward pipes.